build.sbt

```diff
@@ -44,8 +43,9 @@ lazy val benchmark = project.in(file("serial-benchmark"))
   .settings(moduleName := "finagle-serial-bencrhmark")
```

Comment:
BTW, what do you think about avoiding `finagle-serial` prefix in folders names but use it for maven?